site2/docs/admin-api-persistent-topics.md

```diff
@@ -314,27 +345,35 @@ It shows detailed statistics of a topic.
 
   -   **ledgers**: The ordered list of all ledgers for this topic holding its messages
 
+      -   **ledgerId**: Id of this ledger
+
+      -   **entries**: Total number of entries belong to this ledger
+
+      -   **size**: Size of messages written to this ledger (in bytes)
+
+      -   **offloaded**: Whether this ledger is offloaded
+
   -   **cursors**: The list of all cursors on this topic. There will be one for every subscription you saw in the topic stats.
 
-  -   **markDeletePosition**: The ack position: the last message the subscriber acknowledged receiving
+      -   **markDeletePosition**: The ack position where the subscriber acknowledges receiving the last message
```

Comment:
The description of `markDeletePosition` is not correct here(not just this PR). The mark delete position cannot be equivalent to the last message of the subscriber since we have `Shared` and `Key_Shared` subscription. 

The `markDeletePosition` represent the all of messages before the `markDeletePosition` are acked.

Reply:
That is a constructive suggestion！